shared/crypto/keypair.go

```diff
@@ -111,6 +115,16 @@ func (kp encKeyPair) GetSeed(passphrase string) ([]byte, error) {
 	return privKey.Seed(), nil
 }
 
+// Derives a new child key using the key as the master
```

Comment:
Just a NIT (but its outside the scope of this PR) that I learnt this this week: proper godoc comments can be found here: https://go.dev/blog/godoc

tl;dr `<FunctionName> <ExplanationWhatFunctionDoes>`

Reply:
This might be a good starter task actually, I will go through the keybase and slips stuff later today and get the comments in order 🤝

Reply:
🤝